app/scenes/ActivityThanks/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import { H1 } from 'native-base';
+import { Button } from 'native-base';
+import { connect } from 'react-redux';
+import { Actions } from 'react-native-router-flux';
+import {
+  BodyText,
+  Heading,
+} from '../../components/core';
+
+
+const styles = StyleSheet.create({
+  box: {
+    padding: 20,
+    paddingTop: 40,
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: 'white',
+  },
+});
+
+
+const onPressStart = () => {
+  Actions.applet_details();
+}
+
+class ActivityThanks extends Component {
+  render() {
+    return (
+      <View style={{...styles.box}}>
+        <Heading>Thanks!</Heading>
+        <BodyText>
+          We've saved your answers
+        </BodyText>
+        <Button onPress={onPressStart} full rounded style={{marginTop: 20}}>
+          <Text style={{color: 'white'}}>Close</Text>
+        </Button>
+      </View>
+    );
+  }
+}
+
+const mapStateToProps = state => ({
+  // currentResponse: currentResponsesSelector(state),
+  // authToken: authTokenSelector(state),
+});
+
+const mapDispatchToProps = {
+  // setAnswer,
+  // completeResponse,
+};
+
+export default connect(mapStateToProps, mapDispatchToProps)(ActivityThanks);
```

Comment:
No need to do a `connect` if there are no dispatch or state. Suggest just doing `export default ActivityThanks`

Reply:
fixed!

Eventually I'd like the Thanks to include a message like "We'll ask you to take this again on ___" if the activity has another scheduled event. Is this info stored in the state somewhere?